openreview/conference/builder.py

```diff
@@ -636,6 +641,22 @@ def create_blind_submissions(self, force=False, hide_fields=[]):
 
         return blinded_notes
 
+    def setup_venue_papers(self):
+        if not self.submission_stage.due_date or self.submission_stage.due_date < datetime.datetime.now():
+            # Due date is in the past
+            if self.submission_stage.double_blind:
+                # Double Blind venue
+                self.create_blind_submissions()
+            else:
+                # Single Blind or Open venue
+                self.create_paper_groups(authors=True, reviewers=True, area_chairs=True)
+            
+            reveal_authors = False
+            if not self.submission_stage.double_blind:
+                reveal_authors = True
+            self.create_withdraw_invitations(reveal_authors=reveal_authors, reveal_submission=False)
```

Comment:
I think there are a few tests that are calling this directly, it would be nice to remove the calls and see if the tests are still passing.